src/test/java/org/openquantumsafe/SigTest.java

```diff
@@ -69,4 +104,12 @@ private static Stream<String> getEnabledSigsAsStream() {
         return enabled_sigs.parallelStream();
     }
 
+//    /**
```

Comment:
Is there a reason for this commented code block?

Reply:
It was added because only ML-DSA can use a context at the moment.   I suppose in the future SLH-DSA and others might make use of it, and they would need testing as well.